yt_dlp/extractor/facebook.py

```diff
@@ -529,6 +554,7 @@ def extract_metadata(webpage):
                     webpage, 'view count', default=None)),
                 'concurrent_view_count': get_first(post, (
                     ('video', (..., ..., 'attachments', ..., 'media')), 'liveViewerCount', {int_or_none})),
+                'channel_follower_count': follower_count,
```

Comment:
based this on your traversal path above, haven't tested it myself

```suggestion
                'channel_follower_count': get_first(post, (
                    'user', 'profile_header_renderer', 'user', 'profile_social_context', 'content', ..., 'text',
                    lambda k, v: k == 'text' and v.endswith('followers'), {parse_count})),
```